main.go

```diff
@@ -122,4 +128,14 @@ func main() {
 		setupLog.Error(err, "problem running manager")
 		os.Exit(1)
 	}
+
+	setupLog.Info("starting console")
```

Comment:
I think we should call this controller above where we are calling other controllers in the same file.